networkx/generators/directed.py

```diff
@@ -305,3 +308,83 @@ def _choose_node(G, distribution, delta, psum):
         
     return G
 
+
+def random_k_out_graph(n, k, alpha, self_loops=True):
```

Comment:
For now I'll add the `seed=None` keyword argument, like the other functions that use randomness, and if a different strategy is decided upon, I can update this pull request.